Live/LiveEventWithDVR/Program.cs

```diff
@@ -79,522 +74,375 @@
+    // See the overview - https://learn.microsoft.com/azure/media-services/latest/live-event-concept
 
-    #region NewLiveEvent
     var liveEventData = new MediaLiveEventData(mediaServicesAccount.Get().Value.Data.Location)
     {
-        Description = "Sample LiveEvent from .NET SDK sample",
+        Description = "Sample LiveEvent from the .NET SDK sample",
         UseStaticHostname = true,
         // 1) Set up the input settings for the Live event...
         Input = new LiveEventInput(streamingProtocol: LiveEventInputProtocol.Rtmp)
         {
             StreamingProtocol = LiveEventInputProtocol.Rtmp,
             AccessToken = "acf7b6ef-8a37-425f-b8fc-51c2d6a5a86a",
-            KeyFrameIntervalDuration = TimeSpan.FromSeconds(2)
+            KeyFrameIntervalDuration = TimeSpan.FromSeconds(2),
+            IPAllowedIPs =
+            {
+                new IPRange()
+                {
+                    Name = "AllowAllIpV4Addresses",
+                    Address = IPAddress.Parse("0.0.0.0"),
+                    SubnetPrefixLength = 0
+                },
+                new IPRange()
+                {
+                    Name = "AllowAllIpV6Addresses",
+                    Address = IPAddress.Parse("0::"),
+                    SubnetPrefixLength = 0
+                }
+            }
         },
         // 2) Set the live event to use pass-through or cloud encoding modes...
         Encoding = new LiveEventEncoding()
         {
             EncodingType = LiveEventEncodingType.PassthroughStandard
         },
-        // 3) Set up the Preview endpoint for monitoring based on the settings above we already set.
-        Preview = liveEventPreview,
+        // 3) Set up the Preview endpoint for monitoring
+        Preview = new LiveEventPreview
+        {
+            IPAllowedIPs =
+            {
+                new IPRange()
+                {
+                    Name = "AllowAllIpV4Addresses",
+                    Address = IPAddress.Parse("0.0.0.0"),
+                    SubnetPrefixLength = 0
+                },
+                new IPRange()
+                {
+                    Name = "AllowAllIpV6Addresses",
+                    Address = IPAddress.Parse("0::"),
+                    SubnetPrefixLength = 0
+                }
+            }
+        },
+        // 4) Set up more advanced options on the live event. Low Latency is the most common one. Set
+        //    this to Default or Low Latency. When using Low Latency mode, you must configure the Azure
+        //    Media Player to use the quick start heuristic profile or you won't notice the change. In
+        //    the AMP player client side JS options, set -  heuristicProfile: "Low Latency Heuristic
+        //    Profile". To use low latency optimally, you should tune your encoder settings down to 1
+        //    second GOP size instead of 2 seconds.
+        StreamOptions =
+        {
+            StreamOptionsFlag.LowLatency
```

Comment:
Should we be using this flag still? Or should we be using LL v2 with a 720p transcoding channel ?

Reply:
I think that can be a separate change.